scriptmodules/emulators/jzintv.sh

```diff
@@ -44,8 +52,9 @@ function build_jzintv() {
     mkdir -p jzintv/bin
     cd jzintv/src
 
+    isPlatform "rpi" && local extra='EXTRA=-DPLAT_LINUX_RPI'
```

Comment:
You need to declare the variable as local for all platforms nor just for the RPI.

Reply:
I'd like to challenge this. Works also fine when the initial clause is not true.

Reply:
It may work, but it will mean the variable is used undeclared for non RPI platforms and if the variable had something in it in a parent function that data would be included. So this is not correct.

Reply:
Ah. Now I understand. Corrected it.